src/sidebar/util/group-list-item-common.js

```diff
@@ -10,6 +10,9 @@ export function orgName(group) {
   return group.organization && group.organization.name;
 }
 
+/**
+ * @param {import("../services/analytics").Analytics} analytics
+ */
 export function trackViewGroupActivity(analytics) {
```

Comment:
This function isn't actually used anywhere in the codebase any more. We could just remove it and associated tests.